### ISSUE TITLE
modd_turb_cloud suppression

### ISSUE THREAD

**Issue by SebastienRietteMTO** [UMR-CNRM/PHYEX#13]
Would it be possible to move the modd_turb_cloud keys into modd_turbn?
This would allow a better flexibility (activation of cloud mixing length modification in several models) and the suppression of the KMI argument of turb.

**Comment by QuentinRodier** [comment]
Solved in commit e43073f
options are added in modd_turbn. CTURBLEN_CLOUD (name change). 
KMI is removed from argument of turb and from PHYEX%MISC as it is only used in lima call (aro_adjust_lima.F90) => I hard-coded KMI=1 in the call.

A slight problem unsolved since the changed in modd_turbn.F90 is that I have a running crash (invalid memory reference) at the writting of NAM_TURBn in modd_turbn.F90 lines : 
IF(IPRINT>=3) THEN
  WRITE(UNIT=KLUOUT,NML=NAM_TURBn)
ENDIF

I skipped the problem by increasing the IPRINT condition from 1 to 3 but this must be solved but I do not know how ? need help on that.
